scripts/travis/couch-config

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash -eu
 
-if ! [[ "${USE_COUCHDB-}" = "true" ]]; then
-	echo '[medic] skipping couch-config'
-	exit 0
-fi
-
 # Create couchdb system tables (this has to be done manually on couchdb 2.0)
-curl -X PUT 'http://127.0.0.1:5984/{_users,_replicator,_global_changes,_metadata,admins}'
-curl -X PUT 'localhost:5984/medic'
+curl -X PUT 'http://localhost:5984/{_users,_replicator,_global_changes,_metadata,admins,medic-test}'
+
+echo '[medic] creating admin'
+curl -X PUT 'http://localhost:5984/_node/_local/_config/admins/admin' -d '"pass"'
```

Comment:
How did we get away without doing this before!@?!

Reply:
It was being done in grunt but this is clearer IMO.